common/model/bundle.c

```diff
@@ -12,6 +12,60 @@
 
 static UT_icd bundle_transactions_icd = {sizeof(iota_transaction_t), 0, 0, 0};
 
+static retcode_t validate_signature(bundle_transactions_t const *const bundle,
+                                    trit_t const *const normalized_bundle,
+                                    bool *const is_valid) {
+  iota_transaction_t *curr_tx = NULL, *curr_inp_tx = NULL;
+  Kerl address_kerl, sig_frag_kerl;
```

Comment:
we should pass the Kerl instances as an argument to this method

Reply:
passing the Kerl from `bundle_validator` or the caller of `bundle_validator`?

Reply:
actually, I don't see a problem about Kerl instances, this's a private method.

Reply:
Having two Kerl instances here + the one in the caller method will mean 3 * 1.4kB memory vs 2 * 1.4kB if we can pass in an instance here.

Reply:
The `bundle_validator` needs 3 Kerl instances whatever pass it or not.